libs/partners/nvidia-trt/langchain_nvidia_trt/llms.py

```diff
@@ -238,6 +238,7 @@ def _invoke_triton(self, model_name, inputs, outputs, stop_words):
             inputs=inputs,
             outputs=outputs,
             request_id=request_id,
+            enable_empty_final_response=self.client.get_model_config(model_name).config.model_transaction_policy.decoupled
```

Comment:
I want to improve it here. Maybe cache it or use it as model_ready call above. I also don't know how much this grpc code is null proof.